static/javascripts/angular/route/templates/accomodation.html

```diff
@@ -0,0 +1,42 @@
+<div>
+    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 center-block" style="margin-bottom: 80px;">
+        <label  class="control-label label_color col-lg-12 col-md-12 col-sm-12 col-xs-12">Accomodation Type</label>
+                <select  class="form-control">
+                    <option value="" disabled selected>Select your accomodation type</option>
+                    <option>&nbsp;&nbsp;Hotel</option>
+                    <option>&nbsp;&nbsp;Hostel</option>
+                    <option>&nbsp;&nbsp;Camping</option>
+                    <option>&nbsp;&nbsp;Home Rental</option>
+                    <option>&nbsp;&nbsp;Couchsurfing</option>
```

Comment:
@cansintartici Why do not we use a ng-repeat for this option. We can say that we will have a array containing <code>[Hotel, Hostel, Camping, Home Rental, Couchsurfing]</code> and we will use ng-repeat for <code>option</code>